zerver/openapi/zulip.yaml

```diff
@@ -5480,6 +5481,1326 @@ paths:
+                              type: integer
+                              description: |
+                                The highest message id of the conversation.
+                            user_ids:
+                              type: array
+                              items:
+                                type: integer
+                              description: |
+                                The list of users other than the current user in the private message
+                                conversation(empty for PMs to self).
+                        description: |
+                          Present if `recent_private_conversations` is present in `event_types`.
+
+                          A dictionary where each entry describes one of the user's
+                          recent private conversations.
+                      subscriptions:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/Subscriptions"
+                        description: |
+                          Present if `subscription` is present in `event_types`.
+
+                          A array of dictionaries where each dictionary describes one of the
+                          user's subscription to a stream.
+                      unsubscribed:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/BasicStream"
+                        description: |
+                          Present if `subscription` is present in `event_types`.
+
+                          A array of dictionaries where each dictionary describes one of the
+                          streams the user has unsubscribed from.
+                      never_subscribed:
+                        type: array
+                        items:
+                          allOf:
+                            - $ref: "#/components/schemas/BasicStream"
+                            - properties:
+                                stream_weekly_traffic:
+                                  type: integer
+                                  nullable: true
+                                  description: |
+                                    The average number of messages sent to the stream in recent weeks,
+                                    rounded to the nearest integer.
+
+                                    Null means the stream was recently created and there is
+                                    insufficient data to estimate the average traffic.
+                                subscribers:
+                                  type: array
+                                  items:
+                                    type: integer
+                                  description: |
+                                    A list of user IDs of users who are also subscribed
+                                    to a given stream. Included only if `include_subscribers` is `true`.
```

Comment:
Is it really accurate that `never_subscribed` has properties that `unsubscribed` doesn't?